src/azul/service/manifest_service.py

```diff
@@ -916,8 +916,7 @@ def manifest_content_hash(self) -> int:
 
     def file_name(self, object_key, base_name: Optional[str] = None) -> str:
         if base_name:
-            file_name_prefix = unicodedata.normalize('NFKD', base_name)
-            file_name_prefix = re.sub(r'[^\w ,.@%&\-_()\\[\]/{}]', '_', file_name_prefix).strip()
+            file_name_prefix = base64.urlsafe_b64encode(base_name.encode()).decode()
```

Comment:
Ideally we wouldn't need to decode again and use the bytes as the tag value. If tag value must be a string, decode ASCII.

Reply:
Tags values must be strings.

```
...
2022-04-15 14:19:26,105 INFO    MainThread: Tagging object 'manifests/e9309ecd-3f7b-56b1-b54f-72a203c802a7.64610d37-a501-5b3a-a7e8-2b7f10e37144.tsv' with {'TagSet': [{'Key': 'azul_file_name', 'Value': b'TW91c2UgTWVsYW5vbWEgMjAyMi0wNC0xNSAxNC4xOS50c3Y='}]}
2022-04-15 14:19:26,689 DEBUG   MainThread: Tearing Down Data

Error
Traceback (most recent call last):
...
  File "/Users/daniel/repo/azul2/src/azul/service/manifest_service.py", line 1061, in write
    self.storage.put_object_tagging(object_key, tagging)
  File "/Users/daniel/repo/azul2/src/azul/service/storage_service.py", line 212, in put_object_tagging
    self.client.put_object_tagging(Bucket=self.bucket_name,
  File "/Users/daniel/repo/azul2/.venv/lib/python3.8/site-packages/botocore/client.py", line 357, in _api_call
    return self._make_api_call(operation_name, kwargs)
  File "/Users/daniel/repo/azul2/.venv/lib/python3.8/site-packages/botocore/client.py", line 648, in _make_api_call
    request_dict = self._convert_to_request_dict(
  File "/Users/daniel/repo/azul2/.venv/lib/python3.8/site-packages/botocore/client.py", line 696, in _convert_to_request_dict
    request_dict = self._serializer.serialize_to_request(
  File "/Users/daniel/repo/azul2/.venv/lib/python3.8/site-packages/botocore/validate.py", line 293, in serialize_to_request
    raise ParamValidationError(report=report.generate_report())
botocore.exceptions.ParamValidationError: Parameter validation failed:
Invalid type for parameter Tagging.TagSet[0].Value, value: b'TW91c2UgTWVsYW5vbWEgMjAyMi0wNC0xNSAxNC4xOS50c3Y=', type: <class 'bytes'>, valid types: <class 'str'>
```